keras/preprocessing/image.py

```diff
@@ -346,6 +346,7 @@ class ImageDataGenerator(object):
         featurewise_std_normalization: divide inputs by std of the dataset.
         samplewise_std_normalization: divide each input by its std.
         zca_whitening: apply ZCA whitening.
+        zca_epsilon: epsilon for ZCA whitening. Default is 1e-6.
```

Comment:
As stated in the issue: A suitable epsilon depends on the scaling.

As [this](http://ufldl.stanford.edu/wiki/index.php/Data_Preprocessing) website states:
> Tip: If your data has been scaled reasonably (e.g., to [0,1]), start with epsilon = 0.01 or epsilon = 0.1. 

This epsilon is special to zca_whitening and might not be suitable for other cases where you need epsilon.